DESIGN.rst

```diff
@@ -0,0 +1,54 @@
+Code organization
+=================
+
+The repository keeps multiple versions of fake-juju, each building
+against the associated real-juju code.
```

Comment:
I might be wrong, but I believe "corresponding" has a slightly different meaning in English than in Italian. I changed this with "relevant".